doc/man/jose-fmt.1.adoc

```diff
@@ -1,163 +1,168 @@
-jose-fmt(1) -- Converts JSON between serialization formats
-==========================================================
+jose-fmt(1)
+===========
+:doctype: manpage
 
-## SYNOPSIS
+== NAME
 
-`jose fmt` [OPTIONS]
+jose-fmt - Converts JSON between serialization formats
```

Comment:
Why do the command names have hyphens? Those aren't the commands.

Reply:
I'll fix the hyphens, it's caused by the script I've created for the conversion. Didn't catch that during my review.